### PR TITLE
Merge latest changes

### PR DESCRIPTION
**Pull request by natedooley**
Changes include adding pylint, fixing small errors, styling, and typos, updating the route from /profile/<uid> to /user/<uid> (which is what old profiles used and what other services try and link to), and image fallback to Github. Before this gets merged I'll fix image cropping.

### CODE REVIEW

File: profiles/__init__.py [hunk 1]
```diff
@@ -68,47 +77,48 @@ def results(uid=None, info=None):
...
+    						    info=info,
+    						    title=group_desc,
+    						    members=_ldap_get_group_members(_group))
 
 
 @app.route("/update", methods=["POST"])
 @auth.oidc_auth
 @before_request
 def update(uid=None, info=None):
     if request.method == "POST"  and 'photo' in request.files:
```
Comment (devinmatte) [inline profiles/__init__.py]: There's no need to check if "POST" if you've only allowed "POST" in your methods list

File: profiles/static/js/croppie.js [hunk 2]
```diff
@@ -0,0 +1 @@
...={viewport:{width:100,height:100,type:"square"},boundary:{},orientationControls:{enabled:!0,leftClass:"",rightClass:""},resizeControls:{width:!0,height:!0},customClass:"",showZoomer:!0,enableZoom:!0,enableResize:!1,mouseWheelZoom:!0,enableExif:!1,enforceBoundary:!0,enableOrientation:!1,enableKeyMovement:!0,update:function(){}},q.globals={translate:"translate3d"},u(q.prototype,{bind:function(e,t){return O.call(this,e,t)},get:function(){var e=H.call(this),t=e.points;return this.options.relative&&(t[0]/=this.elements.img.naturalWidth/100,t[1]/=this.elements.img.naturalHeight/100,t[2]/=this.elements.img.naturalWidth/100,t[3]/=this.elements.img.naturalHeight/100),e},result:function(e){return P.call(this,e)},refresh:function(){return function(){W.call(this)}.call(this)},setZoom:function(e){L.call(this,e),h(this.elements.zoomer)},rotate:function(e){D.call(this,e)},destroy:function(){return function(){var e,t,n=this;n.element.removeChild(n.elements.boundary),e=n.element,t="croppie-container",e.classList?e.classList.remove(t):e.className=e.className.replace(t,""),n.options.enableZoom&&n.element.removeChild(n.elements.zoomerWrap),delete n.elements}.call(this)}}),e.Croppie=window.Croppie=q});
```
Comment (devinmatte) [inline profiles/static/js/croppie.js]: Why are we storing a vendored library in our repository? We can get it from a cdn: https://cdnjs.com/libraries/croppie
Ideally we would move to using something like yarn or npm to grab out packages, but a cdn is preferable to storing vendored code

File: profiles/__init__.py (hunk 1 above)
Comment (natedooley) [reply profiles/__init__.py]: fixed them

File: profiles/static/js/croppie.js (hunk 2 above)
Comment (natedooley) [reply profiles/static/js/croppie.js]: done it

**Pull request merged**